spec/models/user_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  before(:all) do
+    @user = create(:user)
```

Comment:
@sinpai this was mentioned in previous PRs, so be confident about things we discussed before:

![image](https://user-images.githubusercontent.com/3635774/37456059-93f58fbc-284f-11e8-9c7e-f7025b6cde49.png)